irohad/ametsuchi/mutable_storage.hpp

```diff
@@ -74,7 +72,7 @@ namespace iroha {
        */
       virtual bool apply(
           const shared_model::interface::Block &block,
-          MutableStoragePredicateType<decltype(block)> function) = 0;
+          MutableStoragePredicateType function) = 0;
```

Comment:
In `check()` method name of parameter `function` is omitted, here not. Please, make them consistent